tests/api_fastapi/core_api/routes/public/test_dags.py

```diff
@@ -193,7 +193,7 @@ class TestPatchDag(TestDagEndpoint):
         "query_params, dag_id, body, expected_status_code, expected_is_paused",
         [
             ({}, "fake_dag_id", {"is_paused": True}, 404, None),
-            ({"update_mask": ["field_1", "is_paused"]}, DAG1_ID, {"is_paused": True}, 400, None),
+            ({"update_mask": ["field_1", "is_paused"]}, DAG1_ID, {"is_paused": True}, 200, True),
```

Comment:
This should still raise `400`. Only `is_paused` is updatable, anything else should error and warn the client that the field is not editable.